### PR TITLE
Loans and reservations status

### PR DESCRIPTION
**Pull request by yepzdk**
https://platform.dandigbib.org/issues/5388

Add component with Loans and Reservation status lists

State with data:
Screenshot 2022-07-28 at 12 39 54

Empty state:
Screenshot 2022-07-28 at 12 39 47

### CODE REVIEW

File: src/stories/Blocks/status-userprofile/statusUserprofile.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+export const StatusUserprofile = (props: StatusUserprofileProps) => {
+  const { loansTitle, loans, reservationsTitle, reservations } = props;
+  return (
+    <div className="status-userprofile">
+      <div className="status-userprofile__column">
+        <LinkFilters filters={loansTitle} />
+
+        {loans.length ? (
+          loans.map((item, index) => (
+            <div key={index} className="status-userprofile__loans-list-item">
```
Comment (Adamik10) [inline src/stories/Blocks/status-userprofile/statusUserprofile.tsx:27]: Won't react complain over this in the browser console? I think I remember getting an error in the browser when I used index as the key for mapped elements. 
Funnily enough, I actually JUST made a function to generate IDs out of indexes for exactly this purpose, but it's in my current open, yet un-merged PR (#113).

File: src/stories/Blocks/status-userprofile/statusUserprofile.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          <ListEmpty text="Du har i øjebilkket 0 fysiske lån" />
+        )}
+      </div>
+      <div className="status-userprofile__column">
+        <LinkFilters filters={reservationsTitle} />
+
+        {reservations.length ? (
+          reservations.map((item, index) => (
+            <div
+              key={index}
```
Comment (Adamik10) [inline src/stories/Blocks/status-userprofile/statusUserprofile.tsx:46]: same question applies here for this key

**Review by Adamik10 — APPROVED** [review]
Very very nice 🏆 great job! I have 1 question in the comments, but it's approved.

File: src/stories/Blocks/status-userprofile/statusUserprofile.tsx (hunk 1 above)
Comment (yepzdk) [reply src/stories/Blocks/status-userprofile/statusUserprofile.tsx:27]: It doesn't seem to be the case with errors in the console. But thanks for the heads up.

**Pull request merged**